packages/form-validations/index.html

```diff
@@ -252,7 +252,7 @@ <h2 class="docs-Heading">Custom error messages</h2>
           <label for="custom-errors-checkbox" class="Checkbox" data-spirit-validate>
             <input type="checkbox" id="custom-errors-checkbox" name="custom-errors-checkbox" class="Checkbox__input" required data-spirit-required-message="You must accept the terms and conditions" />
             <span class="Checkbox__text">
-              <span class="Checkbox__label Checkbox__label--required">I accept the terms and conditions</span>
+              <label class="Checkbox__label Checkbox__label--required">I accept the terms and conditions</label>
```

Comment:
Semantically, it is a wrong nested label inside the label. 

Shouldn't it be the root element `div`?